README.md

```diff
@@ -194,7 +194,7 @@ Ferrum::Browser.new(options)
   * `:proxy` (Hash) - Specify proxy settings, [read more](https://github.com/rubycdp/ferrum#proxy)
   * `:save_path` (String) - Path to save attachments with [Content-Disposition](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition) header.
   * `:env` (Hash) - Environment variables you'd like to pass through to the process
-
+  * `:mobile` (Boolean) - Specify whether to enable mobile emulation and touch UI.
```

Comment:
I don't think we need flag for browser instantiation.

Reply:
My intent was to expose this via driver registration, as per Selenium.  

I have a draft PR open for Cuprite that uses this; see e.g.:

https://github.com/rubycdp/cuprite/pull/292/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94

```
Capybara.register_driver(:cuprite_mobile) do |app|
  options = { mobile: true }
  options.merge!(inspector: true) if ENV["INSPECTOR"]
  options.merge!(logger: StringIO.new) if ENV["CI"]
  options.merge!(headless: false) if ENV["HEADLESS"] == "false"
  Capybara::Cuprite::Driver.new(app, options)
end
```